ElevatorSim/src/elevatorsim/floor/FloorController.java

```diff
@@ -58,13 +86,28 @@ public void run() {
 		}
 	}
 	
+	public void processEvents() {
```

Comment:
Does this need to be public?

Reply:
setting to private

Reply:
Made the iter_2_floor_refactoring branch to fix this concern